### DISCUSSION TITLE
Construction Timelapse Issue recording at all

### DISCUSSION THREAD

**Discussion by adleraugen** [gopro/labs#345] [Q&A]
Hi
We have bought 2 GoPros for documenting our house building process. Unfortunately both GoPros (Hero 10 / Hero 11) does not start with the record. We are using the following QR-Code Template: https://gopro.github.io/labs/control/set/?cmd=!06:30NmTp.300!S!43260E!U!1R&title=Daily%20Timelapse%20and%20Upload

The strange thing is when I get back to them after some days there have still the daily schedule (prompting for cancel the timer when powering on the GoPro).

I tried it also with and without the batteries installed, did not make any difference.

Do you have any idea how can I setup the GoPro to finally record my daily construction progress? It's really sad that this is not working as it should, since we bought basically 2 GoPros just for that.

Just want to add the things that I have tried already:
- switched to different Modes (2min Intervall / 5min / etc)
- disabled powering off on the GoPros
- i am currently trying just to record 24/7 just to make sure, I have at least some footage (manually started).

Thanks in advance for your help!

Regards
Michael

**Comment by dnewman-gpsw** [comment]
Recording 24/7 is always the safest, and is a production feature and thoroughly tested.  I've run a camera for three months this way. 

The Labs timelapse scripting is designed for extending battery to the maximum, with the tradeoff of stability.
 
I'm not sure what this means: "does not start with the record."  That script absolute should start.  However there is a bug on HERO10 Labs that is making this type of command unreliable.  There is an untested patch for Labs (H10) posted about a month on this forum, here is a refresh for that link: https://www.dropbox.com/transfer/AAAAAFlTQ8vpofFqfchhxV5MlkWijDyyn9PoChwnQKAlQUV4OEtvzOw  An official Labs refresh for HERO10 or 11 is not expected for months.

On HERO11, this works.  I testing on both HERO10 and 11, and this command worked (I tested that it at least started correctly.) 

mT command is expecting your camera has a TimeLapse Video preset, that is the default, but some users change this to Photo timelapse (then mT will not work, need mTP.)

To learn more about an error, your can sure the MISC/qrlog.txt , which a log for Labs commands and behaviors.

**Reply by adleraugen** [reply]
Thanks for your answer Newman-gpsw! Can you tell me how can I verify that I have a Timelapse video preset configured on my GoPros? Is it just simply configure one (resolution, interval, lens)? Does the interval on the preset be the same as the one with the GoPro labs code?

Regarding your question about the starting:
The camera have not started regarding for days. But I you power on it it keeps saying: planned regarding xxx)

The strange thing also is: I have also tested the codes at home, and they worked as they should. Could weather conditions influence it? But it's spring here in Switzerland, so no more extreme cold conditions....

Thanks in advance!

**Reply by dnewman-gpsw** [reply]
Only need to have a timelapse preset with the Photo selected.  Everything can be done through QR codes.
![image](https://github.com/gopro/labs/assets/10856084/891c083d-f69b-4521-8f3f-581227e89c43)

Need to see the log file.

**Reply by adleraugen** [reply]
Thanks for the clarification regarding the presets (but I am using video, not photo).

Attached you will find the older log files. I was on construction site yesterday to try another run
[qrlog.txt](https://github.com/gopro/labs/files/11448616/qrlog.txt)

**Reply by dnewman-gpsw** [reply]
Which camera is this from?  Which firmware version?  Should not be behaving like that, it is not sleeping correctly.

**Reply by adleraugen** [reply]
It's the GoPro 10 with the newest firmware v1.50.70. The strange thing is that the GoPro 11 (also with newest firmware) does not work with the same setting. Since it is harder for me to get there (its mounted on the window of the neighbor), I will check tomorrow.

**Reply by dnewman-gpsw** [reply]
I will need to see the log file from the HERO11.

**Comment by PixelMafiaProductions** [comment]
I have exactly the same issue with both of my goPro 10s. They started the first recording, but didnt upload it (what can be the problem of bad wifi on the roof of my customers location) and then it didnt start recording today. Will get the log tomorrow, as i set a new qr code now without upload.

Else i will try your newer Labs Software, as i only have the "normal" Labs on them right now.
